app/controllers/api/v1/worksheets_controller.rb

```diff
@@ -3,6 +3,11 @@ class Api::V1::WorksheetsController < Api::V1::BaseController
   before_action :set_and_authorize_work_group, only: %i[new create index]
   before_action :set_and_authorize_worksheet, only: %i[show edit update]
 
+  def all
```

Comment:
I would suggest the name "student_worksheets", but right now the code of this method is exactly the same as that of index, so I'm not sure what you are trying to accomplish

Reply:
The normal index method gets all the worksheets assigned to the student in the context of a specific work group, but I need to get all the worksheets assigned to the student period, regardless of work group. That's what this method does.

Reply:
(It's not including in the before action calls at the top, which means it doesn't return just the worksheets associated with the classroom and work group)